examples/Bunch/include/simulation_defines/param/gasConfig.param

```diff
@@ -47,6 +47,13 @@ namespace gasProfiles
 {
 
 PMACC_STRUCT(GaussianCloudParam,
+    /** Gas Formula:
+     *     exponent = |globalCellPos - center| / sigma
+     *     density = e^[ gasFactor * exponent^gasPower ]
+     */
+    (PMACC_C_VALUE(float_X, gasFactor, -1.0))
```

Comment:
I did not want to change your definition of your `Bunch` example (this line).
I would definitely change the default, good point.

Do you want to change it in the Bunch example as well?